src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const bigDiscount = 50;
```

Comment:
The variable name 'bigDiscount' is not clear enough. Rename it to 'largeDiscountAmount' to indicate it's the discount amount for rentals of 7 days or more. 💡